core-bundle/src/EventListener/PrettyErrorScreenListener.php

```diff
@@ -204,22 +211,30 @@ private function renderTemplate(string $template, int $statusCode, ExceptionEven
     }
 
     /**
-     * @return array<string,string|int>
+     * @return array<string,mixed>
```

Comment:
Why not `array<string,string|int|bool|\Throwable>`?

Reply:
This is what phpstan told me to write…